### DISCUSSION TITLE
Disabling Input field for the Input Date Picker

### DISCUSSION THREAD

**Discussion by EvolvedMeta** [ing-bank/lion#2044] [Ideas] [answered]
For small screen devices, We have seen the use input field on the date-picker sometimes is not very convenient. So, can we extend existing functionality for datepicker and allow developer to disable the input field and so the user can select the date through calendar only.

**Comment by gerjanvangeest** [comment, marked as answer]
Hi @EvolvedMeta, thanks for starting this discussion.

Although it is easier for you to select a date via the calendar, that might not be the case for all users. Some people prefer to stick with the UI they are used to, like a keyboard, and have problems understanding new UI. So disabling a feature might do more harm then good.

You could add this feature to your own extension layer, but i don't think we will add it to our core.
